NeuroID/src/reactNativeLib/java/com/neuroid/tracker/callbacks/NIDActivityCallbacks.kt

```diff
@@ -124,6 +124,16 @@ class NIDActivityCallbacks() : ActivityLifecycleCallbacks {
                     attrs = attrJSON
                 )
             )
+
+
+        val currentActivityName = activity::class.java.name
+        registerTargetFromScreen(
+            activity,
+            registerTarget = true,
+            registerListeners = true,
+            activityOrFragment = "activity",
+            parent = currentActivityName
+        )
```

Comment:
@kishisaka-nid  - Unsure if we need this in my PR or if your work will cover it, but left it in because of our session yesterday

Reply:
Yeah we need this otherwise the registrations don't occur and we don't get edited events.

Reply:
@kishisaka-nid  - I removed this because we are now going to be calling forceStart on every page and if the user does go in and out of the app I didn't want to re-register things and maybe have another duplicate listener issue